### PR TITLE
Cookbook geodesic

### PR DESCRIPTION
**Pull request by F-Loyer**

### CODE REVIEW

File: data/cookbook/calculate-geodistance-between-points-spherical/00-stdlib.ml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+packages: []
+discussion: |
+  - The Earth surface can be approximated by a 6371 km sphere. Calculating distance between 2 points can be done with the Haversine formula. See [this page](https://www.movable-type.co.uk/scripts/latlong.html).
+---
+let pi = 3.14159265356279
```
Comment (cuihtlauac) [inline data/cookbook/calculate-geodistance-between-points-spherical/00-stdlib.ml]:
```suggestion
```

File: data/cookbook/calculate-geodistance-between-points-spherical/00-stdlib.ml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+let deg2rad angle =
+  angle /. 180. *. pi
```
Comment (cuihtlauac) [inline data/cookbook/calculate-geodistance-between-points-spherical/00-stdlib.ml]:
```suggestion
  angle /. 180. *. Float.pi
```

File: data/cookbook/calculate-geodistance-between-points-wgs84/00-stdlib.ml [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
+packages: []
+discussion: |
+  - The Earth can be approximated by an ellipsoid defined by the WGS84 reference system. The [Vincety formulae](http://www.movable-type.co.uk/scripts/latlong-vincenty.html) can be used to calculate distance in this system. The following code is translated from [this site](http://walter.bislins.ch/bloge/index.asp?page=WGS84+JavaScript+Module).
+---
+
+let pi = 3.14159265356279
```
Comment (cuihtlauac) [inline data/cookbook/calculate-geodistance-between-points-wgs84/00-stdlib.ml]:
```suggestion
```

File: data/cookbook/calculate-geodistance-between-points-wgs84/00-stdlib.ml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+let deg2rad angle =
+  angle /. 180. *. pi
```
Comment (cuihtlauac) [inline data/cookbook/calculate-geodistance-between-points-wgs84/00-stdlib.ml]:
```suggestion
  angle /. 180. *. Float.pi
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Review by F-Loyer — COMMENTED** [review]
Proposals (change pi to Float.pi) commited.

**Comment by nilsbecker** [comment]
I think it would avoid potential confusion for newcomers to avoid the syntax `function(arg + arg2)` and instead use the convention of spaces between function and arguments throughout `function (arg1 + arg2)`. Also, choose a uniform style for operator spacing `f a b /. 2.` instead of `f a b/.2` for instance. Maybe use automatic formatting?

**Pull request closed without merging**